.github/ISSUE_TEMPLATE/New_release.md

```diff
@@ -14,36 +14,34 @@ This issue is to provide visibility on the progress of the release process of Gu
 ## Resources
 
 -   📖 Read: [Gutenberg Release Process](https://developer.wordpress.org/block-editor/contributors/code/release/)
--   📖 Read: [Leading a Gutenberg Plugin Release](https://codep2.wordpress.com/2021/07/22/leading-a-gutenberg-plugin-release/)
```

Comment:
This link should be removed as it's inaccessible for the community

Reply:
It has been removed :)